src/App.js

```diff
@@ -1,23 +1,31 @@
-import logo from './logo.svg';
-import './App.css';
+import { useState } from "react";
+import { searchRecipes, getRecipe } from "./api";
+import RecipeSearch from "./component/RecipeSearch";
+import RecipeList from "./component/RecipeList";
+import DisplayRecipe from "./component/DisplayRecipe";
```

Comment:
`RecipeSearch` and `DisplayRecipe` are not "proper" component name. Usually we name them in such a way that they are proper nouns, for example `RecipeSearchForm` and `RecipeDetail`.